Units/simple-robot.d/expected.tags

```diff
@@ -4,18 +4,16 @@
 !_TAG_PROGRAM_AUTHOR	Universal Ctags Team	//
 !_TAG_PROGRAM_NAME	Universal Ctags	/Derived from Exuberant Ctags/
 !_TAG_PROGRAM_URL	https://ctags.io/	/official site/
-!_TAG_PROGRAM_VERSION	0.0.0	/e5be352/
+!_TAG_PROGRAM_VERSION	0.0.0	/f6247b0/
```

Comment:
Pseudo tags (tags started from `!_`) should be removed from expected.tags. I should write this earlier. Sorry.